src/lib/inputleap/ServerApp.h

```diff
@@ -118,11 +118,9 @@ class ServerApp : public App {
 
 // configuration file name
 #if SYSAPI_WIN32
-#define USR_CONFIG_NAME "input-leap.sgc"
-#define SYS_CONFIG_NAME "input-leap.sgc"
+#define CONFIG_NAME "input-leap.sgc"
 #elif SYSAPI_UNIX
-#define USR_CONFIG_NAME ".input-leap.conf"
-#define SYS_CONFIG_NAME "input-leap.conf"
+#define CONFIG_NAME "input-leap.conf"
```

Comment:
Not sure if we should keep a `SYS` config name...

Reply:
Both names are the same so no i don't think we need to store a `sys` config name

Reply:
OK, thanks for the clarification.